src/bin/add/main.rs

```diff
@@ -1,7 +1,13 @@
 //! `cargo add`
 #![warn(
-    missing_docs, missing_debug_implementations, missing_copy_implementations, trivial_casts,
-    trivial_numeric_casts, unsafe_code, unstable_features, unused_import_braces,
+    missing_docs,
+    missing_debug_implementations,
+    missing_copy_implementations,
+    trivial_casts,
+    trivial_numeric_casts,
+    unsafe_code,
+    unstable_features,
+    unused_import_braces,
```

Comment:
I'm using `rustfmt +nightly` and am getting this formatting. Should I override the behavior or is it okay?